content/recipes/applied-examples/eubopen.md

```diff
@@ -0,0 +1,51 @@
+(fcb-eubopen)=
+# FAIRification of High-Content Screening data using Bioimage Archive
+
+````{panels_fairplus}
+:identifier_text: TODO
+:identifier_link: TODO
+:difficulty_level: 3
+:recipe_type: applied_example
```

Comment:
```suggestion
:maturity_level: 1
:maturity_indicator: 1
```

NOTE: the values for the maturity level and indicator are examples. would need to be added.